src/app/textbook/components/textbook-content/textbook-content.component.scss

```diff
@@ -46,6 +51,11 @@
   justify-content: space-between;
 }
 
+.tab-link {
+  color: royalblue;
```

Comment:
If it is a common color you can add css variable

Reply:
Done.